.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[env]
+RUST_TEST_THREADS = "1"
```

Comment:
why?

Reply:
yeah, and i think this is partly the same reason we can't have [TSC snapshot](https://github.com/denoland/deno/pull/27987) anymore right?